src/components/app/MenuButtons/MetaInfo.js

```diff
@@ -53,6 +53,12 @@ class MetaInfoPanelImpl extends React.PureComponent<Props> {
     const { profile, symbolicationStatus, resymbolicateProfile } = this.props;
     const isSymbolicated = profile.meta.symbolicated;
 
+    const isSymbolicationUnsupported =
+      profile.meta.symbolicationNotSupported === true;
```

Comment:
Let's try to use the "positive" (non-negated) way of expressing these things in as many places as possible. For example, here, I'd do the following:

```js
const supportsSymbolication = profile.meta.symbolicationNotSupported !== true;
```

As for the profile.meta flags, using the negated terms there makes sense for compatibility. But I think it would be easier to read to use non-negated terms there, too. Maybe we can accumulate a bunch of these flags over the next few PRs, and then do another clean-up PR on top of that which would bump the profile version and write an upgrader, changing all the field names.